openmetadata-service/src/main/java/org/openmetadata/service/resources/events/ChangeEventResource.java

```diff
@@ -41,12 +42,12 @@
 import org.openmetadata.service.util.EntityUtil;
 import org.openmetadata.service.util.ResultList;
 
-@Path("/v1/events")
-@Api(value = "Events resource", tags = "events")
+@Path("/v1/changeEvents")
```

Comment:
Let's keep the previous /v1/events instead of /v1/changeEvents

Reply:
done